### PR TITLE
Integrations from Kubecon San Diego

### PR DESCRIPTION
**Pull request by timothyhinrichs**

### CODE REVIEW

**Comment by timothyhinrichs** [comment]
Will squash these into a single commit before merging, but left them separate for ease? of review.  Let me know if you'd rather I squash them together for review as well.  Didn't quite keep them separate for the main track of talks because there weren't any new integrations for those.

**Comment by timothyhinrichs** [comment]
@patrick-east @tsandall @ashutosh-narkar can someone approve this if it looks good?

File: docs/website/data/integrations.yaml [hunk 1]
```diff
@@ -522,6 +669,27 @@ organizations:
...
+    link: https://www.tripadvisor.com/
+  chef:
+    name: Chef
+    link: https://www.chef.io/
+  buoyant:
+    name: Buoyant
+    link: https://buoyant.io/
+  netflix:
+    name: Netflix
+    link: https://www.netflix.com/
```
Comment (ashutosh-narkar) [inline docs/website/data/integrations.yaml]: Can we add `capitalone` and `yelp`

Comment (timothyhinrichs) [reply docs/website/data/integrations.yaml]: Done

Comment (timothyhinrichs) [reply docs/website/data/integrations.yaml]: Done

**Pull request merged**